feature-libs/storefinder/core/facade/store-finder.service.ts

```diff
@@ -56,6 +56,7 @@ export class StoreFinderService implements OnDestroy {
 
   /**
    * Returns observable for store's entities
+   * TODO: The return value of thism ethod signature is wrong, should be StoreFinderSearchPage.
```

Comment:
[MINOR REMARK] 
1. let's create a ticket for it instead of putting a raw TODO
2. I believe SonarQube checks in CI won't let you merge with new `TODO` comments included, so I guess you'll need to remove it or rephrase not to be `TODO` 😅